src/Storefront/Resources/app/storefront/src/plugin/cookie/cookie-configuration.plugin.js

```diff
@@ -425,21 +427,13 @@ export default class CookieConfiguration extends Plugin {
         this.closeOffCanvas();
     }
 
-    acceptAllCookies() {
-        this.openOffCanvas(() => {
-            this._handleAcceptAll();
-        });
-    }
-
     /**
-     * Event handler for the 'Allow all'-button in the off canvas view.
-     *
      * This will set and refresh all registered cookies.
      *
      * @private
      */
-    _handleAcceptAll() {
-        const allCookies = this._getCookies('all');
+    handleAcceptAll(offCanvas = null) {
```

Comment:
In case this method is now public intentionally, please change the `@private` tag accordingly.